### PR TITLE
ceph-osd: enable CentOS extras using yum-config-manager

### PR DESCRIPTION
**Pull request by gites**

### CODE REVIEW

File: roles/ceph-osd/tasks/pre_requisite.yml [hunk 1]
```diff
@@ -5,12 +5,14 @@
     state: present
   when: ansible_os_family == 'Debian'
 
+- name: Install yum utils
```
Comment (leseb) [inline roles/ceph-osd/tasks/pre_requisite.yml]: Please don't use capital letters.
Why do we want to install yum-utils?

This doesn't really look like a mandatory package for Ceph.
If there is anything else you'd like to see, it has to be part of your own infrastructure playbook.

👎

Comment (gites) [inline roles/ceph-osd/tasks/pre_requisite.yml]: yum-config-manager is provided by yum-utils package, so it need to be present.

**Comment by leseb** [comment]
please rebase

**Comment by leseb** [comment]
please rebase :)

**Comment by gites** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by leseb** [comment]
jenkins test this please

**Comment by ktdreyer** [comment]
The commit message tells us what the commit *does* but not *why*?

**Comment by leseb** [comment]
@gites please see @ktdreyer's comment.

**Comment by ktdreyer** [comment]
It would be good to mention https://github.com/ceph/ceph-ansible/issues/1133 in the commit message

**Pull request closed without merging**